packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-utils.js

```diff
@@ -157,6 +159,7 @@ const FlowStageUtils = {
       sellerConfig: sellerConfig || SELLER_CONFIG.primary,
       customer: this.createRandomCustomerDetails(includeAllOptionalCustomerDetails ?? false),
       orderItemCriteriaList,
+      describeFeatureRecord,
```

Comment:
This will now be input into all the FlowStage constructors. It's used only by the Test Interface Action Flow Stage for now